src/config/accounts.c

```diff
@@ -958,6 +958,10 @@ accounts_get_login_status(const char* const account_name)
     if (g_strcmp0(setting, "last") == 0) {
         status = accounts_get_last_status(account_name);
     }
+
```

Comment:
I don't think we need an if here, right?

Reply:
yes, g_free() and free() handle NULL pointers. I just made in the same way as in other places in this file.

Reply:
I usually try to get rid of them when I encounter them :)